app/models/container_image_registry.rb

```diff
@@ -4,6 +4,8 @@ class ContainerImageRegistry < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
   has_many :containers, :through => :container_images
+  has_one :container_service
+  has_many :container_groups, :through => :container_service
```

Comment:
this was in the requirement and I display them as well.
As I see it if there is a service that creates a registry but no pods
it's important for the user to see since it will explain why the registry is actually down.